src/locales/en.json

```diff
@@ -559,6 +561,7 @@
     "type_aria_label": "Select filter type",
     "type_aws": "AWS",
     "type_empty": "Choose source type",
+    "type_gcp": "GCP",
```

Comment:
This looks like dead code to me. I don't see type_aws, type_ocp, etc. used anywhere.

It appears these 3 keys can be removed safely

Reply:
Done